CHANGELOG.md

```diff
@@ -20,6 +20,7 @@
 - Added: [#5701](https://github.com/ethereum/aleth/issues/5701) Outputs ENR text representation in admin.nodeInfo RPC.
 - Added: [#5705](https://github.com/ethereum/aleth/pull/5705) Istanbul support: EIP 1108 Reduce alt_bn128 precompile gas costs.
 - Added: [#5707](https://github.com/ethereum/aleth/pull/5707) Aleth waits for 2 seconds after sending disconnect to peer before closing socket.
+- Changed: [#5735](https://github.com/ethereum/aleth/pull/5735) Tests <= ConstantinopleFix moved to `-t LegacyTests suite` in the testeth
```

Comment:
```suggestion
- Changed: [#5735](https://github.com/ethereum/aleth/pull/5735) `testeth -t GeneralStateTests` and `testeth -t BlockchainTests` commands now don't run the tests for the forks before Istanbul. They are moved to a separate `LegacyTests` suite, to run them use the commands `testeth -t LegacyTests/Constantinople/GeneralStateTests` and ``testeth -t LegacyTests/Constantinople/BlockchainTests -- --all`
```
please check the commands there ^

also better move it to be the last of `Changed` (to keep them sorted by id)